tasklist/app/views/people/tasks.html.erb

```diff
@@ -0,0 +1,15 @@
+<h1> Assigned Tasks </h1>
+
+<h2> <%= @person.name %></h2>
+<ul>
+  <% @person.tasks.each do |task| %>
+    <li>
+    <%= task.name %> <br>
+      <% if task.date_completed == nil %>
```

Comment:
It is preferred to use the `.nil?` method as opposed to check for equivalence to `nil`